### DISCUSSION TITLE
How to really make sure vd asks before quitting?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1289] [Q&A] [answered]
A few times I lost some (small and unimportant) work because I pressed `Q` too often and vd exited without saving...

I have set `options.quitguard = True` in my rc file, but it doesn't seem to better the situation. I also have updated to version 2.8.

 Do I do something wrong?

**Comment by saulpw** [comment]
`quitguard` protects against losing changes; it won't have any effect on sheets which don't have any modifications.  Are you losing work in some other way?

**Reply by halloleo** [reply]
Well, `quitguard` doesn’t seem to do this for me: I open a CSV, add a column via the `=` command, go to the collumn sheet, press (by a accident) twice `q` — and have exited vd!

**Reply by anjakefala** [reply]
Can you make a bug report, please @halloleo? =)

**Reply by halloleo** [reply]
Done as  #1290.

**Reply by anjakefala** [reply]
Thank you!

**Comment by halloleo** [comment, marked as answer]
Works correctly from version 2.8. See #1290.

**Reply by anjakefala** [reply]
Fixed on `develop`! =) Will be in 2.9.
